assessments/projects/spiffe-spire/outline.md

```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
 
 _Maturity_
 
 (Examples)
-- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exclusive list and it’s likely that due to the secrecy of a security framework other organizations are also SPIFFE and SPIRE users privately.
+- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exhaustive list, due to the sensitivity surrounding exposure of security frameworks in use it is likely other organizations are also SPIFFE and SPIRE users privately.
 - SPIFFE and SPIRE are being used by numerous other companies, both large and small, to build higher layer products and services. This includes Decipher Technology Studios, F5 Networks, HashiCorp, Intel, Google, IBM, Tigera, VMware, and many others.
 - The SPIFFE project has 5 owners from 5 different organizations. 
 - The SPIRE project has 7 owners from 2 different organizations.
 
 
 ## Summary
 
-**Design**: SPIFFE/SPIRE have a streamlined and simple design. It covers many use cases of identity without exposing unnecesary complexity to the user. The SPIFFE 'Workload API' provides a great way for services and orchestration systems to integrate with. The the pluggable design for attestation, key management, databases, etc... allows for ease of extensibility.
+**Design**: Covering many use cases of identity without exposing unnecesary complexity to the user, SPIFFE/SPIRE provide a streamlined and simple design to interact with. The SPIFFE '[Workload API](https://spiffe.io/spiffe/concepts/#spiffe-workload-api)' provides a great way for services and orchestration systems to integrate. This pluggable design for attestation, key management, databases, and more allows for easy extensibility.
 
 **Analysis**: The project, as a security provider, has done due diligence in security and threat modeling. The security workflow is evident, and the project is in the correct direction to further improve its security process and CI verification. 
-The project fulfills the role of a layer below the orchestration framework, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
+The project fulfills a needed role layer below the orchestration framework layer, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
 
 All questions from reviewers were addressed in [self-assessment](self-assessment.md)
 with non-critical issues captured as issues and noted below.
 
 ## Recommendations
 
-### Recommendations to the project team
+### Recommendations to the Project Team
```

Comment:
https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md

This should follow sentence case

Reply:
corrected